www/src/scss/controller.scss

```diff
@@ -517,4 +554,4 @@ ul {
 }
 
 @import 'foundation/foundation', 'animations', 'footer', 'forms', 'header', 'helpers', 'search',
-  'share-sync', 'settings', 'notifications';
+'share-sync', 'settings', 'notifications';
```

Comment:
Should have new line at the end.

https://eslint.org/docs/latest/rules/eol-last